src/client/pages/OfferNew/Checkout/index.tsx

```diff
@@ -263,6 +271,10 @@ export const Checkout: React.FC<CheckoutProps> = ({
     })
       .then(({ data }) => {
         if (data?.signQuotes?.__typename === 'FailedToStartSign') {
+          if (data?.signQuotes.errorCode === 'MANUAL_REVIEW_REQUIRED') {
+            setShowFailModal(true)
+            return
+          }
```

Comment:
Does this code only apply to NO users?

Otherwise, it seems like we don't give the other users any indication something went wrong. Since the `SignFailModal` is only rendered in NO.

However, if this error code is NO-specific we would not need to conditionally render the `SignFailModal`, right? That would remove some market specific logic which I think is the way forward!

Reply:
Yeah I think this error-code is specifically for if credit-checks fail in NO, and if so I can remove the market-check for the SignFailModal. Will double check with backend!